server/api/util.go

```diff
@@ -20,17 +20,14 @@ import (
 	"net/http"
 	"net/url"
 
+	"github.com/pingcap/pd/v4/server/cluster"
 	"github.com/pkg/errors"
 )
 
-// dialClient used to dial http request.
-var dialClient = &http.Client{
-	Transport: &http.Transport{
-		DisableKeepAlives: true,
-	},
-}
-
 var (
+
+	// dialClient used to dial http request.
+	dialClient        = cluster.DialClient
```

Comment:
The client in cluster package has a timeout. Does the client in API needs it?

Reply:
I checked it and it only used in a redirect for the scheduler handler, I think it's ok to timeout.

Reply:
If we use API to get all regions and the region number is large, will it break the process of getting regions?

Reply:
the client in the user, not in PD.

Reply:
got